packages/api/src/command/webhook/webhook.ts

```diff
@@ -195,12 +214,13 @@ export const sendPayload = async (
   }
 };
 
-export const sendTestPayload = async (url: string, key: string): Promise<boolean> => {
+export const sendTestPayload = async (url: string, key: string, cxId: string): Promise<boolean> => {
   const ping = nanoid();
   const payload: WebhookPingPayload = {
     ping,
   };
-  const res = await sendPayload(payload, url, key, DEFAULT_TIMEOUT_SEND_TEST_MS);
+  const timestamp = "0:00";
```

Comment:
timestamp added so header can be attached

Reply:
Can we use the actual timestamp like we do on regular messages?